doc/changelog/08-vernac-commands-and-options/18785-janno-fix-dnet-proj.rst

```diff
@@ -0,0 +1,5 @@
+- **Added:**
+  new vernacular command :cmd:`Hint Projections` similar to :cmd:`Hint Constants`
+  but applying to projections
```

Comment:
```suggestion
  :cmd:`Hint Projections` command that sets the transparency flag for projections
  for the specified hint databases.
```

"Added" already implies "new"
Better just to say what it does rather than compare it to an existing command.

Reply:
Done.